index.html

```diff
@@ -15,11 +15,15 @@
             <a href="#" class="logo">
 					My Logo
 				</a>
-            <img src="./assets/images/Union.svg" alt="Menu" class="mobile-menu">
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                  </div>
             <ul class="menu">
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
+                <li><a href="#about" class="nav-link">About</a></li>
+                <li><a href="#contact" class="nav-link">Contact</a></li>
```

Comment:
- [ ] Please you mobile menu is missing the following features, kindly implement them: 

  - When the user clicks (or taps) the hamburger button, the mobile menu appears.
  - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
  - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
  - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Reply:
Changes have been made to the index.html file to fix the mobile menu unresponsive error.
This was due to the wrong link to the main.js file

Reply:
well resolved! ✔️